client/src/components/Predictions.js

```diff
@@ -1,23 +1,67 @@
 import React, { Component } from "react";
 import styled from "styled-components";
-import Row from "./Row";
+// import bennyboy from "../assets/profilePhotos/bennyboy.jpeg";
 
 const StyledPredictionsDiv = styled.div`
   position: fixed;
   top: 75px;
 `;
 class Predictions extends Component {
+  createTable = () => {
+    let table = [];
+    const { users, matches } = this.props;
+    // Outer loop to create parent
+    for (let i = 0; i < users.length + 1; i++) {
```

Comment:
everytime that you are looping over something's length, then it is better to use `.forEach` cause its cleaner and easier to read

Reply:
no changes, as discussed